amd/ubi9-python-3.9/Dockerfile

```diff
@@ -0,0 +1,49 @@
+# Install ROCm AMD from:
+# https://github.com/ROCm/ROCm-docker/blob/master/dev/Dockerfile-centos-7-complete
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}
+LABEL name="odh-notebook-rocm-python-3.9" \
+      summary="ROCm Python 3.9 base image for ODH notebooks" \
+      description="ROCm Python 3.9 builder image based on UBI9 for ODH notebooks" \
+      io.k8s.display-name="ROCm Python 3.9 base image for ODH notebooks" \
+      io.k8s.description="ROCm Python 3.9 builder image based on UBI9 for ODH notebooks" \
+      authoritative-source-url="https://github.com/opendatahub-io/notebooks" \
+      io.openshift.build.commit.ref="main" \
+      io.openshift.build.source-location="https://github.com/opendatahub-io/notebooks/tree/main/amd/ubi9-python-3.9" \
+      io.openshift.build.image="quay.io/opendatahub/workbench-images:rocm-ubi9-python-3.9"
+
+USER 0
+WORKDIR /opt/app-root/bin
+
+ARG ROCM_VERSION=6.1
+ARG AMDGPU_VERSION=6.1
+
+# Enable epel-release repositories
+# TODO: Remove the CentOS Stream repos once the ROCm dependency are in UBI
+RUN yum -y install https://mirror.stream.centos.org/9-stream/BaseOS/x86_64/os/Packages/centos-stream-repos-9.0-26.el9.noarch.rpm  \
```

Comment:
Is there any particular reason for not combining sequential `RUN` commands into a single one? I understand each `RUN` creates one extra layer, which extends build time and increases the container image size.

Reply:
If your `yum install` installs new files into `/etc/yum.repos.d/`, then there need to be multiple yum invocations, one to install the new .repo files, and the other one to use the repos. Still, it could still be done in a single Dockerfile `RUN`.

If the only reason for multiple RUNs is readability, would it make sense to put all these RUNs into a single bash script and run that from the Dockerfile? Another reason may be faster and simpler local development that makes use of layer caching.

This image is IMO ok, when there are multiple `yum installs` in some other images where each downloads the yum cache, deletes it and the end, and the yum running right after downloads cache again, that's something to fix!

Reply:
>Is there any particular reason for not combining sequential RUN commands into a single one? I understand each RUN creates one extra layer, which extends build time and increases the container image size.

Rightly pointed out, corrected it to the single RUN command